.eslintrc

```diff
@@ -112,7 +128,17 @@
         "@typescript-eslint/no-inferrable-types": "off",
         "@typescript-eslint/no-explicit-any": "off",
         "@typescript-eslint/no-this-alias": "off",
-        "@typescript-eslint/no-unused-vars": "off"
+        "no-unused-vars": "off",
+        "@typescript-eslint/no-unused-vars": [
+          "error",
+          {
+            "vars": "all",
+            "args": "none",
+            "varsIgnorePattern": "^_.+",
+            "ignoreRestSiblings": true,
+            "destructuredArrayIgnorePattern": "^_"
```

Comment:
The options are almost the same with the JS counterpart, but the TS version has `destructuredArrayIgnorePattern` otherwise, there will be some files that fails.